### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by mohitdhatrak**
Change log: added login and signup pages

Screenshots: 

image

image

### CODE REVIEW

**Review by pnchinmay — APPROVED** [review]
Looks good :100:

File: Styles/navigation.css [hunk 1]
```diff
@@ -47,7 +48,7 @@
     font-size: 1.5rem !important;
```
Comment (Rishi05051997) [inline Styles/navigation.css:48]: Is any specific reason for important

**Review by Rishi05051997 — COMMENTED** [review]
Looks Good

File: Styles/navigation.css (hunk 1 above)
Comment (mohitdhatrak) [reply Styles/navigation.css:48]: yes, google icons not allowing any change in size, tried few things didn't work. Only using it for google icons size over ride no other place.

**Pull request merged**